Tribler/community/triblerchain/community.py

```diff
@@ -189,8 +189,14 @@ def get_node(self, public_key, nodes, total_up=None, total_down=None):
             if total_up and total_down:
                 return {"public_key": public_key, "total_up": total_up, "total_down": total_down}
             else:
-                total_traffic = self.persistence.total_traffic(public_key)
-                return {"public_key": public_key, "total_up": total_traffic[0], "total_down": total_traffic[1]}
+                binary_public_key = public_key
+                total_traffic = self.persistence.total_traffic(binary_public_key)
+                latest_block = self.persistence.get_latest(binary_public_key)
+                if latest_block:
```

Comment:
Are we using the latest block in the trustchain or are we using the sum of all edges in the aggregates database?